.github/workflows/test.yml

```diff
@@ -47,8 +47,9 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - run: sudo apt-get install pandoc
-      - run: python setup.py sdist
       - run: pip install --upgrade pip
+      - run: pip install setuptools
```

Comment:
will this cause any trouble if running for version < 3.12?

Reply:
No. In Python < 3.12, setuptools [is installed automatically](https://github.com/python/cpython/issues/95299), so installing it explicitly here has no effect.

(It is actually only required in Python 3.12 because we are installing commcare-export using setup.py in the next line. setuptools is not required when commcare-export is installed with `pip install -e .`.)